OpenEdXMobile/res/values/strings.xml

```diff
@@ -482,7 +482,7 @@
 
     <!-- Registration Screen -->
     <!-- Create account agreement. Followed by list of agreement names like "End User License Agreement" -->
-    <string name="by_creating_account">By creating an account, you agree to the </string>
+    <string name="by_creating_account">By creating an {platform_name} account, you agree to the </string>
```

Comment:
@catong, what do you think of this? The string reads, 

By creating an {platform_name} account, you agree to the

where {platform name} is replaced in real time depending on the configuration. What should we do about "an" vs "a"? Should we just put it as "a" universally? There will be a grammatical error in our app if we do.